### ISSUE TITLE
Aria labels for pagination buttons

### ISSUE THREAD

**Issue by edenhernandez-suse** [rancher/dashboard#8114]
## Context
The pagination buttons are icon buttons without any text description:

![image](https://user-images.githubusercontent.com/120640736/217542040-5b1e4742-2f6e-4128-89b5-e3a29a221ff4.png)

## The bug
Screen-readers read them as "_button_".

## Expected Result
These buttons should have an aria-label that explains what they do. We can use the following rules to make the pagination buttons accessible: [Accessible Pagination](https://www.a11ymatters.com/pattern/pagination/)

Part of #7968

**Comment by a-arias** [comment]
**TESTS**

Rancher | v2.12-d7f62116eb48fb060f02a10c8b01164048d9eeda-head
Dashboard | master https://github.com/rancher/dashboard/commit/01a2d88593058b075515a71a7e6d68e07b0da411


Fix verified ✅ , added arial label to the buttons:

<img width="1396" alt="Image" src="https://github.com/user-attachments/assets/7ab2f2c3-342c-493a-9241-82b6f5c66415" />
<img width="1086" alt="Image" src="https://github.com/user-attachments/assets/b2ad90da-0c96-44bb-9061-a751e9de7d24" />
